dd-smoke-tests/log-injection/build.gradle

```
@@ -357,6 +352,12 @@ generateTestingJar('Flogger', 'Slf4jToLogbackLatest',
 
 tasks.withType(Test).configureEach {
   dependsOn tasks.withType(ShadowJar)
+
+  if (System.getenv("RUNTIME_AVAILABLE_PROCESSORS_OVERRIDE") != null) {
+    maxParallelForks = Integer.valueOf(System.getenv("RUNTIME_AVAILABLE_PROCESSORS_OVERRIDE"))
+  } else {
+    maxParallelForks = Math.max(1, (Runtime.getRuntime().availableProcessors() / 2) as int)
+  }
```

Could you check if this will be more Groovy-style:
```suggestion
def override = System.getenv("RUNTIME_AVAILABLE_PROCESSORS_OVERRIDE")
maxParallelForks = override?.toInteger() ?: Math.max(1, (Runtime.runtime.availableProcessors() / 2).toInteger())
```
And also would be nice to have some comment here why we need to limit maxParallelForks?